libs/community/tests/unit_tests/agent_toolkits/test_imports.py

```diff
@@ -3,6 +3,7 @@
 EXPECTED_ALL = [
     "AINetworkToolkit",
     "AmadeusToolkit",
+    "AzureAiServicesToolkit",
```

Comment:
There are analogous tests `tests.unit_tests.tools.test_imports` and `tests.unit_tests.tools.test_public_api` that may also need to be updated.

Reply:
I am not familiar with testing tbh, but let me know if I can help somehow?